packages/lucide-react/src/createLucideIcon.ts

```diff
@@ -8,9 +8,10 @@ export type SVGAttributes = Partial<SVGProps<SVGSVGElement>>
 export interface LucideProps extends SVGAttributes {
   size?: string | number
   absoluteStrokeWidth?: boolean
+  ref?: RefObject<SVGSVGElement>;
 }
 
-export type LucideIcon = ForwardRefExoticComponent<LucideProps & RefAttributes<SVGSVGElement>>
+type LucideIcon = ForwardRefExoticComponent<LucideProps & RefAttributes<SVGSVGElement>>;
```

Comment:
Sorry maybe I was not clear, I mean like this. We should keep the export.
```suggestion
export type LucideIcon = ForwardRefExoticComponent<LucideProps>;
```

Reply:
Or is this not working?